packages/instantsearch.js/src/components/CurrentRefinements/CurrentRefinements.tsx

```diff
@@ -72,6 +72,7 @@ const CurrentRefinements = ({
 
               <button
                 className={cssClasses.delete}
+                type="button"
```

Comment:
This should be non breaking change, we have a test that ensures we do not submit a parent form when deleting a current refinement.